src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,45 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final int MAX_LENGTH_OF_STORAGE = 10;
+    private final Object[] keysStorage;
```

Comment:
It` s better to generify your storage arrays;
```suggestion
    private final K[] keysStorage;
```